src/control/server/ctl_storage_rpc.go

```diff
@@ -339,14 +354,14 @@ func (c *ControlService) doFormat(i *IOServerInstance, reformat bool, resp *ctlp
 		resp.Crets = append(resp.Crets, nvmeResults...) // append this instance's results
 
 		if nvmeResults.HasErrors() {
-			c.log.Error(msgFormatErr)
-			return nil // don't continue if we can't format NVMe
+			c.log.Errorf(msgFormatErr, srvIdx)
```

Comment:
Is there any information available to help disambiguate which of multiple nvme devices on the machine generated the error?  I see that you're passing the server index, but I'm wondering if there's a name and/or pci address available?